### PR TITLE
New design for Wallet History

### PR DESCRIPTION
**Pull request by MandeepDaffo**
Screenshot_20210412-005334_ECC Wallet

### CODE REVIEW

File: app/src/main/res/layout/fragment_history.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/background_home">
+    android:background="@color/zcashBlack_0">
```
Comment (gmale) [inline app/src/main/res/layout/fragment_history.xml:8]: The design was a little chopped off so it's hard to see that this background was intended to be the same. This line edit can be reverted.

**Review by gmale — CHANGES_REQUESTED** [review]
@Mandeepbhalothia 
Wow. Thanks for the quick turn around time and even putting in work over the weekend. I appreciate the efforts! Thanks for investing your time. I reviewed the code and also paired with the designer to get his feedback.

First, this is pretty much exactly what was expected and more!

Next, here is a quick summary of his feedback:
- Blue is a new color that will be coming to the app and get heavy use for anything "transparent" (i.e. not shielded, using an address that starts with a `t`)
- For a given history item:
    - the `amount` should be:
        - red if negative
        - yellow if shielded (and positive)
        - white if positive (and not shielded)
        - _Implementation Note: "shielded" here is defined as_ `transaction.toAddress.isShielded()`.
    - the `address` should be:
        - white, unless transparent
    - if the transaction is shielded, everything is yellow, like this (to indicate this is preferred):
    - 
    - the left-side bar should be:
        - blue if to or from a transparent address
        - yellow if to or from a shielded address
        - gray if pending (matching the date/time color)
- The designer thought the right-aligned memo icon was a nice touch because it's always in the same place
- The header area will probably change but he wants to make clear designs for that rather than just giving feedback. So what you have for now is great.

File: app/src/main/res/layout/fragment_history.xml (hunk 1 above)
Comment (Mandeepbhalothia) [reply app/src/main/res/layout/fragment_history.xml:8]: Thanks @gmale for sharing the the description related to colour combinations. Hope now we can cover all combinations!
Yes I will check this background thing also.

**Comment by Mandeepbhalothia** [comment]
@gmale Please review the PR. Design is completed by taking conditions from your above comment. Sorry I don't have access to change this Draft PR to ready for review.

**Comment by gmale** [comment]
The code looks good to me. Testing all the scenarios is a little tricky and time consuming. So for the sake of moving forward, I'll merge this and then if we get any additional feedback from the designer or spot any issues, I'll just open issues for that work.

**Pull request merged**